src/main/java/edu/kis/powp/jobs2d/command/visitor/TransformationRotateVisitorImpl.java

```diff
@@ -0,0 +1,57 @@
+package edu.kis.powp.jobs2d.command.visitor;
+
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.ICompoundCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+
+import java.util.Iterator;
+
+public class TransformationRotateVisitorImpl implements IDriverCommandsVisitor {
+	private int angle;
+	private int x;
+	private int y;
+
+	public TransformationRotateVisitorImpl(int angle) {
+		this.angle = angle;
+	}
+
+	private void rotate(int angle){
```

Comment:
Angle argument is also not necessary. 
Notice that flip() and rotate() could be then replaced by a generic transform method (strategy pattern!)